events.json

```diff
@@ -1,5 +1,22 @@
 [
   {
+    "date": "2017-06-13",
```

Comment:
Would it be possible to add the time here? Something like: 2017-06-13T18:00:00?

Reason: Might be nice to see the time in the first view.